devtools/cli/src/main/java/io/quarkus/cli/update/RewriteGroup.java

```diff
@@ -13,8 +13,8 @@ public class RewriteGroup {
     public boolean dryRun = false;
 
     @CommandLine.Option(order = 2, names = {
-            "--update-recipes-version" }, description = "Use a custom io.quarkus:quarkus-update-recipes version. This artifact contains the base recipes used by this tool to update a project.")
-    public String updateRecipesVersion;
+            "--update-recipes-coords" }, description = "Use a custom quarkus-update-recipes coords (GAV) or just version. This artifact should contains the base recipes used by this tool to update a project.")
```

Comment:
```suggestion
            "--update-recipes-coords" }, description = "Use a custom quarkus-update-recipes group:artifact:version (GAV) or just version. This artifact should contain the base recipes to update a project.")
```

"this tool" means what exactly?